server/src/main/java/org/apache/iotdb/db/query/dataset/AlignByDeviceDataSet.java

```diff
@@ -87,6 +89,7 @@ public AlignByDeviceDataSet(
     // adapting AlignByDevice query for new vector
     super.columnNum = alignByDevicePlan.getMeasurements().size() + 1; // + 1 for 'device'
     this.measurements = alignByDevicePlan.getMeasurements();
+    this.withoutNullColumns = alignByDevicePlan.getWithoutNullColumns();
```

Comment:
add some comments about measurements which indicating that it doesn't contain alias.

Reply:
fix